site/en/docs/web-platform/navigation-api/index.md

```diff
@@ -294,7 +516,7 @@ In fact, it will always be `undefined` in those cases.
     height="320",
     muted="true" %}
   <figcaption>
-    <a href="https://wiry-tricolor-lipstick.glitch.me" target="_blank">Demo of opening from left or right</a>
+    <a href="https://gigantic-honored-octagon.glitch.me/" target="_blank">Demo of opening from left or right</a>
```

Comment:
Not sure this should be changed, now it doesn't match the gif? Hmm.

Reply:
oof, good catch, I pulled the wrong URL from my clipboard. It's supposed to be the same demo but updated for the latest API https://chief-low-specialist.glitch.me/. I'll update it.